go/types/sequence_chunker.go

```diff
@@ -173,7 +173,7 @@ func (sc *sequenceChunker) createSequence() (sequence, metaTuple) {
 	seq := col.sequence()
 	mt := newMetaTuple(sc.vrw.WriteValue(col), key, numLeaves)
 
-	sc.current = []sequenceItem{}
+	sc.current = sc.current[:0]
```

Comment:
Since this is a more general-purpose perf win, I wonder if it should be in a separate PR? @rafael-atticlabs, thoughts?

Reply:
yeah, kalman has a patch outstanding in this area. I'll ask him to do this as a part of that.

Reply:
ok, removing